keras_nlp/models/albert/albert_presets.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ALBERT model preset configurations."""
+
+
+backbone_presets = {
+    "albert_base_en_uncased": {
+        "config": {
+            "vocabulary_size": 30000,
+            "num_layers": 12,
+            "num_heads": 12,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 768,
+            "intermediate_dim": 3072,
+            "dropout": 0.0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
+        "description": (
+            "Base size of ALBERT where all input is lowercased. "
+            "Trained on English Wikipedia + BooksCorpus."
+        ),
+        "weights_url": "https://drive.google.com/uc?export=download&id=1RzTTa8nMcBc84nARvJmHal5SndpKbDUa",
+        "weights_hash": "b83ccf3418dd84adc569324183176813",
+        "spm_proto_url": "https://drive.google.com/uc?export=download&id=1AMSRktn0cEUmHUWZNOUsts1Mlw-R74Qu",
+        "spm_proto_hash": "73e62ff8e90f951f24c8b907913039a5",
+    },
+    "albert_large_en_uncased": {
+        "config": {
+            "vocabulary_size": 30000,
+            "num_layers": 24,
+            "num_heads": 16,
+            "num_groups": 1,
+            "num_inner_repetitions": 1,
+            "embedding_dim": 128,
+            "hidden_dim": 1024,
+            "intermediate_dim": 4096,
+            "dropout": 0,
+            "max_sequence_length": 512,
+            "num_segments": 2,
+        },
+        "preprocessor_config": {},
```

Comment:
Is this needed?

Reply:
I think so: https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/deberta_v3/deberta_v3_tokenizer.py#L133-L138.